txbits/app/service/sql/misc.scala

```diff
@@ -153,6 +153,13 @@ object misc {
     """.stripMargin
   )
 
+  val changeFeesToOneWay = SQL(
```

Comment:
This also changes the fee to 0.5%. It's only for dev as prod would not have permissions but I'd still prefer a more generic `update trade_fees set one_way = true`.